tests/functional/test_new_resolver_errors.py

```diff
@@ -101,32 +101,33 @@ def test_new_resolver_checks_requires_python_before_dependencies(
 ) -> None:
     incompatible_python = f"<{sys.version_info.major}.{sys.version_info.minor}"
 
-    pkg_dep = create_basic_wheel_for_package(
+    pkgdep = create_basic_wheel_for_package(
         script,
-        name="pkg-dep",
+        name="pkgdep",
         version="1",
     )
     create_basic_wheel_for_package(
         script,
-        name="pkg-root",
```

Comment:
Why do we need to change the names here?

Reply:
Not sure, I just cherry picked this change from your original PR. [`0538477` (#13160)](https://github.com/pypa/pip/pull/13160/commits/05384778aa2c984594f46f2edfb06219cbedcb19)

Reply:
From the original PR: [`f3c6b2a` (#11398)](https://github.com/pypa/pip/pull/11398/commits/f3c6b2afcbeb5cd483df2b7164742488b7913ee4)

Reply:
@uranusjr if you'd like, I can revert the name changes, although I'd prefer leaving your commit unchanged. Do you remember why you changed the names? I can definitely take a closer look at your commit, but if you can answer the review comments from Paul, I would appreciate that.

Your original PR is more than two years old, though, so don't worry if you have no idea.